src/psyclone/tests/test_files/gocean1p0/reordered_compute_cu_mod.f90

```diff
@@ -0,0 +1,114 @@
+!> \brief Compute the mass flux in the x direction, cu
```

Comment:
Please add the license/copyright header and extend the comment to say what this file is testing. You'll also need to remove the comment that says what the code originally looked like as that no longer matches the implementation (presumably because this is now a different kernel that doesn't exist in the original NemoLite2D code?).

Reply:
Done